exercises/luhn/canonical-data.json

```diff
@@ -123,7 +115,23 @@
       "expected": true
     },
     {
-      "description": "strings with non-digits is invalid",
+      "comments": [
+        "Convert non-digits to their ascii value and then offset them by 48 sometimes accidently return the correct result.",
+        "This test is designed to avoid that solution."
+      ],
+      "description": "using ascii value for doubled non-digit isn't allow",
```

Comment:
I think for grammar, `allow` should be `allowed` here, and in the other test case as well